### PR TITLE
add wrap docs

### PR DESCRIPTION
**Pull request by pileks**
This PR introduces documentation for the UniV3 wrap itself, which will be later hosted on Wrapscan.

A docs manifest with a basic, but usable readme was added.

### CODE REVIEW

File: v3/wrap/polywrap.docs.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+format: 0.1.0
+description: Interact with the Uniswap V3 contract.
+website: https://polywrap.io
+repository: https://github.com/polywrap/ens
```
Comment (krisbitney) [inline v3/wrap/polywrap.docs.yaml]: https://github.com/polywrap/uniswap

File: v3/wrap/docs/readme.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Integrate
+
+### Step 1: Polywrap Client
+
+In order to integrate the UniV3 wrap into your applications, the first thing you'll need is a Polywrap client. Currently Polywrap has clients available in:
+- JavaScript / TypeScript
+- Python
+- Rust
+- Go
```
Comment (krisbitney) [inline v3/wrap/docs/readme.md]: I don't think the Go client is in a ready state. Does it have an ethers provider plugin?

**Review by krisbitney — APPROVED** [review]

File: v3/wrap/docs/readme.md (hunk 2 above)
Comment (pileks) [reply v3/wrap/docs/readme.md]: This was mainly taken from the README in the UniV3 demo, so I didn't use my brain too much when setting it up.

I'll have it removed.

**Comment by kev-ngo** [comment]
>  Currently Polywrap has clients available in:
> * JavaScript / TypeScript
> * Python
> * Rust
> * Swift

Should we only list the clients that are ready for use?  I.e. JS and Rust

> Or you can keep it type-safe by using Polywrap's codegen like so

Should we only show people the type-safe approach?  Personally, it seems like the approach with the better developer experience for me.

cc @pileks

**Comment by pileks** [comment]
> > Or you can keep it type-safe by using Polywrap's codegen like so
> 
> Should we only show people the type-safe approach? Personally, it seems like the approach with the better developer experience for me.
> 
> cc @pileks

@kev-ngo I'd keep both as there's merit to achieving something quickly. The type-safe approach requires the Polywrap CLI, a `schema.graphql` file, a `polywrap.yaml` file and running the `codegen` command itself. There's a tutorial on how to achieve this, and people who will want to use this more will try it out, but as a 1st step, I personally always just `client.invoke` things.

**Pull request merged**